server.js

```diff
@@ -1,23 +1,97 @@
+//Anza Azam NW4 CYF Chat Server
 const express = require("express");
 const cors = require("cors");
-
 const app = express();
 
 app.use(cors());
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+const uuid = require("uuid4");
 
 const welcomeMessage = {
-  id: 0,
-  from: "Bart",
+  id: uuid(),
+  from: "Bar",
   text: "Welcome to CYF chat system!",
+  timeSent: new Date(),
 };
 
+const messages = [welcomeMessage];
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+//const messages = [welcomeMessage];
+console.log(messages);
 
+//different routes //html form
 app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
+// get all messages
+app.get("/messages", (req, res) => {
+  res.json(messages);
+});
```

Comment:
You could add a 200 status to your valid responses too. There's an example here:  https://www.geeksforgeeks.org/express-js-res-status-function/